src/classes/ADDR_Contact_TDTM.cls

```diff
@@ -231,226 +259,63 @@ public class ADDR_Contact_TDTM extends TDTM_Runnable {
-           !ADDR_Addresses_TDTM.equalsCaseSensitive(con1.Primary_Address_Type__c, con2.Primary_Address_Type__c);
-           
-        if (!isChanged && ADDR_Addresses_TDTM.isStateCountryPicklistsEnabled) {
-            isChanged = 
-                !ADDR_Addresses_TDTM.equalsCaseSensitive(string.valueOf(con1.get('MailingStateCode')), string.valueOf(con2.get('MailingStateCode'))) || 
-                !ADDR_Addresses_TDTM.equalsCaseSensitive(string.valueOf(con1.get('MailingCountryCode')), string.valueOf(con2.get('MailingCountryCode')));
-        }    
-        UTIL_Debug.debug('****Address changed: ' + isChanged);
-        return isChanged;   
     }
     
     /*******************************************************************************************************
-    * @description returns whether the contact's mailing address is empty
-    * @param con1 a Contact
-    * @return boolean. true if the Mailing Address fields are all empty 
-    ********************************************************************************************************/
-    private boolean isContactAddressEmpty(Contact con1) {
-        boolean isEmpty = 
-            con1.MailingStreet == null &&
-            con1.MailingCity == null &&
-            con1.MailingState == null &&
-            con1.MailingPostalCode == null &&
-            // note that we decided to ignore country so that a default value won't create unnecessary address objects
-            (con1.MailingCountry == null || ADDR_Addresses_TDTM.isStateCountryPicklistsEnabled) &&
-            // only test StateCode if picklists enabled.
-            (!ADDR_Addresses_TDTM.isStateCountryPicklistsEnabled || con1.get('MailingStateCode') == null);
-        return isEmpty;   
-           
-    }
-       
-    /*******************************************************************************************************
-    * @description Creates a new address or matches an existing one, for each Contact.
-    * This routine is called both by BeforeInsert/Update and AfterInsert/Update.
-    * when pushing a new default address, we need to do it in the AfterInsert/Update.
-    * otherwise we want to do it in the BeforeInsert/Update so we can update the contacts' info now.
-    * @param listConAll the list of Contacts
-    * @param dmlWrapper to hold the Addresses that need inserting/updating 
-    * @param triggerAction which trigger event (BeforeInsert, AfterInsert, etc.). 
+    * @description
+    * @param
+    * @param 
     * @return void  
     ********************************************************************************************************/
-    private void createAddrFromCon(list<Contact> listConAll, DmlWrapper dmlWrapper,  TDTM_Runnable.Action triggerAction) {
+    private void addrInfoDeleted(List<Contact> listContactsAddrInfoCleared, DmlWrapper dmlWrapper) {
+        List<ID> oldCurrentAddrIDs = new List<ID>();
```

Comment:
Variables should use the `lowerCamelCase` naming convention. Please rename accordingly.